fiatconnect-api.md

```diff
@@ -891,7 +885,8 @@ Each `fiatAccount[FiatAccountTypeEnum]` MUST correspond to a fiat account type t
 `fiatAccount[FiatAccountTypeEnum].fiatAccountSchemas` is a list of objects, where each object represents a fiat account schema that can be used to communicate data
 about the corresponding fiat account type. Each object MUST contain a `fiatAccountSchema` field representing the actual fiat account schema that can be used, and an
 optional `allowedValues` field. The `allowedValues` object is an optional mapping from any number of keys in the selected fiat account schema to values that are allowed for that key.
-This is identical in purpose and function to the `allowedValues` field for KYC schemas, discussed earlier.
+For example, if a server wants to limit the selection for certain fields on the schema, `allowedValues` can contain lists of selectable values for those fields.
+On the client-side, this could be used to render a list of options, for example.
```

Comment:
```suggestion
If a server wants to limit the selection for certain fields on the schema, `allowedValues` can contain lists of selectable values for those fields.
On the client-side, this could be used to render a list of options.
```
Nit: these just seem like details to me, not examples